alembic/env.py

```diff
@@ -0,0 +1,85 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+# this is the Alembic Config object, which provides
+# access to the values within the .ini file in use.
+config = context.config
+
+# Interpret the config file for Python logging.
+# This line sets up loggers basically.
+fileConfig(config.config_file_name)
+
+# add your model's MetaData object here
+# for 'autogenerate' support
+# from myapp import mymodel
+# target_metadata = mymodel.Base.metadata
+# target_metadata = None
+from api.models import db
```

Comment:
Does that actually work? `alembic upgrade head` worked for me only with sys.path.append('.') code.

Reply:
I set PYTHONPATH to a project directory, but it's a common thing to do, didn't know it's not the case here. 
(might now work with Docker, but I didn't have a chance to check :) ) 
Will try to switch to absolute paths.

Reply:
Yes, PYTHONPATH works, but we shoudn't set it each time, better set it once in code.